docs/getting-started.md

```diff
@@ -59,12 +59,12 @@ As you can see above `ng g @scullyio/init:blog` creates the following files for
 - blog.component.spec.ts
 - blog.component.ts
 
-Also, it updated the `app-routing.module.ts` with the new module added. Last but not least we added `scully.config.js` and create a
-folder named `/blog` wich inside have a markdown file. 
+In addition, it updates the `app-routing.module.ts` by adding the new module. Last but not least, it adds `scully.config.js` and creates a
+`/blog` folder, which contains a markdown file. 
 
 ###### `scully.config.js`
 
-Inside this file, we can find all the routes of your app generated automatically from Scully. 
+This file has all the routes in the app generated by Scully.
 
 ```js
 exports.config = {
```

Comment:
@Villanuevand this is the format we add in the schematics for routes! 

/${data.name}/:${data.slug}': {
      type: '${data.type}',
      ${data.slug}: {
        folder: "./${data.name}"
      }
    }

Reply:
Great!  Thanks for share @jorgeucano